LuminAi/src/main/frontend/src/services/PowerService.js

```diff
@@ -0,0 +1,26 @@
+function getUpdatedEnergyDevices() {
```

Comment:
the getUpdatedEnergyDevices function should only be responsible to fetch the data from the backend, please put the code to insert the data into the frontend somewhere else

Reply:
i fixed it